lib/tasks/db.rake

```diff
@@ -5,4 +5,18 @@ namespace :db do
       project.source_repositories.create(url: project[:github_url])
     end
   end
+
+  desc "add slack channel codes to project slack channels join table"
+  task add_project_slack_channels: :environment do
+    include ChannelsList
+    Project.all.each do |project|
+      if CHANNELS[project.slug.to_sym].respond_to? :each
```

Comment:
in principle we could use `Array()` here to simplify this code, but we'll only use it once, so if it's working it's almost not worth changing ...